js/noms/src/value-store.js

```diff
@@ -173,14 +175,24 @@ export class SizeCache<T> {
       }
     }
   }
+
+  drop(hash: Hash): void {
+    const key = hash.toString();
+    const entry = this._cache.get(key);
+    if (entry) {
+      this._size -= entry.size;
```

Comment:
yeah, I had that but in deleting my sweet printf debugging it got nuked. Javascript is...